README.md

```diff
@@ -2,13 +2,12 @@
 
 ```
 git clone https://github.com/vz10/geogit.git
+cd geogit
 ```
 
 If you use Kitematic then run
 ```
-# cd geogit
-# eval "$(docker-machine env default)"
-# docker build -t geogit .
+eval "$(docker-machine env default)"
```

Comment:
Maybe you have to add something like "If you are an idiot without native docker"